algosdk/atomic_transaction_composer.py

```diff
@@ -694,6 +725,7 @@ def simulate(
 
         Args:
             client (AlgodClient): Algod V2 client
+            request (models.SimulateReuqest): SimulateRequest with options in simulation.
```

Comment:
Can you say that the txn groups from the request will be overrwritten by the composer's group?

Also, spelling error ("SimulateReuqest" - > "SimulateRequest")

Reply:
see [pr response](https://github.com/algorand/py-algorand-sdk/pull/469/commits/939e1216a567006d6667ecd622965788719a41cf)